src/bin/polymer-bundler.ts

```diff
@@ -198,6 +201,20 @@ options.inlineScripts = Boolean(options['inline-scripts']);
 options.inlineCss = Boolean(options['inline-css']);
 options.rewriteUrlsInTemplates = Boolean(options['rewrite-urls-in-templates']);
 
+class RedirectionResolver extends UrlResolver {
+  private _prefix: string;
+  constructor(prefix: string) {
+    super();
+    this._prefix = prefix;
+  }
+  canResolve(url: string) {
+    return url.startsWith(this._prefix);
+  }
+  resolve(url: string) {
+    return url as ResolvedUrl;
```

Comment:
Yeah it really is just a resolver that says whether or not the URL being redirected from can actually be resolved or not.  Each one of these instantiated has a companion PrefixedUrlLoader that performs the actual "redirection".

Reply:
I moved it into a class so it could extend the base UrlResolver to get the protected method, since UrlResolver was changed to base class instead of an interface.